hack/quickstart-setup.sh

```diff
@@ -16,18 +16,24 @@
 # limitations under the License.
 #
 
-TOOLS_IMAGE=quay.io/kuadrant/mgc-tools:latest
-TMP_DIR=/tmp/mgc
+export TOOLS_IMAGE=quay.io/kuadrant/mgc-tools:latest
+export TMP_DIR=/tmp/mgc
 
 operatorSdkInDocker() {
   docker run -u $UID -v "${TMP_DIR}:${TMP_DIR}:z" --network mgc -e KUBECONFIG="${TMP_DIR}/kubeconfig" --rm ${TOOLS_IMAGE} -c "operator-sdk $*"
 }
 
+yqInDocker() {
+  docker run -u $UID -v "${TMP_DIR}:${TMP_DIR}:z" --rm ${TOOLS_IMAGE} -c "yq $1 '$2' $3"
+}
```

Comment:
@mikenairn @laurafitzgerald This is far from ideal, but works for the two lines that require `yq` that are used in this script (these commands are in the `kindGenExternalKubeconfig` function in `hack/.kindUtils`). The way args are handled here is because of some issues with quotes used in the `yq` commands being lost when passed to this function. I tried to refactor the commands to work for the binary and a version of this function that used `$*`, but couldn't find a way that worked for both. Any ideas how this could be improved?

Reply:
Id try and changing the entrypoint to the container i.e. `docker run --rm -it -u 1000 -v /tmp/mgc:/tmp/mgc:z --network mgc -e KUBECONFIG=/tmp/mgc/kubeconfig --entrypoint=yq quay.io/kuadrant/mgc-tools:latest` 

``` #!/bin/bash

TOOLS_IMAGE=quay.io/kuadrant/mgc-tools:latest

dockerBinCmd() {
  echo "docker run --rm -it -u $UID -v /tmp/mgc:/tmp/mgc:z --network mgc -e KUBECONFIG=/tmp/mgc/kubeconfig --entrypoint=$1 $TOOLS_IMAGE"
}

KUSTOMIZE_BIN=$(dockerBinCmd "kustomize")
YQ_BIN=$(dockerBinCmd "yq")
OPERATOR_SDK_BIN=$(dockerBinCmd "operator-sdk")

${KUSTOMIZE_BIN} version
${YQ_BIN} --version
${OPERATOR_SDK_BIN} version
${OPERATOR_SDK_BIN} olm status 
```